zmon_worker_monitor/builtins/plugins/appdynamics.py

```diff
@@ -194,7 +192,7 @@ def healthrule_violations(self, application, time_range_type=BEFORE_NOW, duratio
             logger.exception('AppDynamics request failed')
             raise
 
-    def query_logs(self, q='', body=None, size=100, source_type=SOURCE_TYPE_APPLICATION_LOG, duration_in_mins=5):
+    def query_logs(self, q='', body=None, size=100, duration_in_mins=5, raw_result=False):
```

Comment:
It is no longer fixed, changed to be unique per application. I believe depending on custom fields in filtering is very specific and could lead to issues. User can still filter using ``q`` param if required.